examples/standalone/runfile/dynamics.py

```diff
@@ -1,120 +1,152 @@
+import json
+import pathlib
 from argparse import ArgumentParser
+from datetime import datetime
 
-import mpi4py
+import git
+import numpy as np
 import serialbox
 import yaml
+from mpi4py import MPI
 
 import fv3core
 import fv3core._config as spec
 import fv3core.stencils.fv_dynamics as fv_dynamics
 import fv3core.testing
 import fv3gfs.util as util
-from fv3core.testing import GlobalTimer, write_to_json
+from fv3gfs.util import Timer
```

Comment:
Please use `util.Timer` instead, following the pattern for other objects from fv3gfs.util. I would also generally suggest `import fv3gfs.util as fv3util` instead of as `util`, but this is optional.